frontend/js/access.js

```diff
@@ -37,6 +37,7 @@ define([
         PUBLIC: 1,
         SHARED_WITH_ADMIN: 2,
         SHARED_WITH_EVERYONE: 3,
+        SHARED_WITH_GROUP: 4,
```

Comment:
This access level can be removed (this belonged to a draft course context feature which is not included in the merge)